apps/config/local.py

```diff
@@ -28,8 +28,7 @@ class Local(Common):
         "apps.core.authentications.AutoLoginAuthentication",
     ]
 
+    LOCAL_DB_PATH = os.path.join(os.path.split(BASE_DIR)[0], "local_db.sqlite3")
```

Comment:
아래처럼 common에 있는 거를 사용해주시면 되긴 하는데요
중복된 코드로 보이는데 없어도 될거같습니다.
LOCAL_DB_PATH = common.LOCAL_DB_PATH
```suggestion
    LOCAL_DB_PATH = os.path.join(os.path.split(BASE_DIR)[0], "local_db.sqlite3")
```